src/newSeed/stage4.ts

```diff
@@ -71,17 +71,29 @@ export class Stage4 extends BaseStage {
     }
   }
 
-  persistData(): void {
+  setSeedConfigStartDate(): Date {
     // Set the ISO time
     // Get the start and end time
     const startTimes = this.startTime.split(":");
-    const endTimes = this.endTime.split(":");
-    let temp = this.startDate;
+    const temp = this.startDate;
     temp.setHours(Number.parseInt(startTimes[0]), Number.parseInt(startTimes[1]));
     this.seedConfig.seedDetails.startDate = this.dateService.toISOString(this.dateService.translateLocalToUtc(temp));
-    temp = this.endDate;
+    return new Date(this.seedConfig.seedDetails.startDate);
+  }
+
+  setSeedConfigEndDate(): Date {
+    // Set the ISO time
+    // Get the start and end time
+    const endTimes = this.endTime.split(":");
+    const temp = this.startDate;
```

Comment:
really?  have you tested this?

Reply:
Sorry made a mistake cleaning up, all corrected now